BLMM/BLMM.py

```diff
@@ -141,11 +69,13 @@ def _main(argv=None):
     # --------------------------------------------------------------------------------
     
     # Ask for a node for setup
-    cluster.scale(numNodes)
-
+    # Make sure we don't use more nodes than necesary 
+    if numNodes > nb+1 or numNodes > nvb+1:
```

Comment:
This is just adjusting the numNodes to the amount of batches or nvb actually needed just in case the user puts more than necessary